### PR TITLE
Feat/add humanizeBytes and showDeltaFileSizes functions

### PR DESCRIPTION
**Pull request by sebastian2296**

### CODE REVIEW

File: src/main/scala/mrpowers/jodie/DeltaHelpers.scala [hunk 1]
```diff
@@ -527,4 +527,27 @@ object DeltaHelpers {
...
+    })
+    val result: String = resultOption.getOrElse(f"$n%.0f" + " B")
+    result
+  }
+
+  def showDeltaFileSizes(deltaTable: DeltaTable) = {
+    val details: Row = deltaTable.detail().select("numFiles", "sizeInBytes").collect()(0)
+    val (sizeInBytes, numberOfFiles) = 
+      (details.getAs[Long]("sizeInBytes"), details.getAs[Long]("numFiles"))
+    val avgFileSizeInBytes = if (numberOfFiles == 0) 0 else Math.round(sizeInBytes / numberOfFiles)
```
Comment (brayanjuls) [inline src/main/scala/mrpowers/jodie/DeltaHelpers.scala]: This code section seems equal to what deltaFileSizes function, could you try to reuse that other function in here?

File: src/main/scala/mrpowers/jodie/DeltaHelpers.scala [hunk 2]
```diff
@@ -527,4 +527,27 @@ object DeltaHelpers {
...
+    val formatter = java.text.NumberFormat.getIntegerInstance
```
Comment (brayanjuls) [inline src/main/scala/mrpowers/jodie/DeltaHelpers.scala]: I think we can remove this line of code as is not being used.

File: src/main/scala/mrpowers/jodie/DeltaHelpers.scala [hunk 3]
```diff
@@ -527,4 +527,27 @@ object DeltaHelpers {
...
+
+    val humanized_number_of_files = numberOfFiles.toInt
+    val humanized_size_in_bytes = humanizeBytes(sizeInBytes)
+    val humanized_average_file_size = humanizeBytes(avgFileSizeInBytes)
```
Comment (brayanjuls) [inline src/main/scala/mrpowers/jodie/DeltaHelpers.scala]: let's stick with lowerCamelCase for variable naming as it is the convention in scala.

**Review by brayanjuls — CHANGES_REQUESTED** [review]
Good effort! I added some comments and I will finish the review for the unit test later today.

File: src/main/scala/mrpowers/jodie/DeltaHelpers.scala (hunk 1 above)
Comment (sebastian2296) [reply src/main/scala/mrpowers/jodie/DeltaHelpers.scala]: Just implemented the change to reuse deltaFileSizes function. In order to make it work properly I added a type annotation to its' signature (it was inferred as Map[String, AnyVal], which was incompatible with HumanizeBytes input type).

**Comment by brayanjuls** [comment]
LGTM

**Pull request merged**